internal/endtoend/testdata/valid_group_by_reference/postgresql/go/query.sql.go

```diff
@@ -79,7 +79,7 @@ ORDER BY bucket DESC
 type ListMetricsRow struct {
 	Bucket   interface{}
 	CityName sql.NullString
-	Avg      string
+	Avg      float64
```

Comment:
The query is:

```
SELECT time_bucket('15 days', time) AS bucket, city_name, AVG(temp_c)
FROM weather_metrics
WHERE DATE_SUB(NOW(), INTERVAL 6 MONTH)
GROUP BY bucket, city_name
ORDER BY bucket DESC;
```

So I think the float64 is actually correct here - the string was incorrect before.  They are also "both wrong" since the order of the function definitions in `pg_catalog.go` is actually what is determining the types here.